### PR TITLE
add eli5 skill

### PR DESCRIPTION
**Pull request by ktzee**
**Describe the contribution to the taxonomy**
Add skill to explain concepts in a short form and basic terms

**Input given at the prompt**
explain why it rains like I'm 5.

**Response that was received**
Imagine there's a big cloud in the sky. The cloud is made up of tiny droplets of water. Now, when the sun shines on the cloud, the water droplets get warmer and start to evaporate, which means they turn into tiny water vapor particles.

But as the cloud gets bigger and bigger, the water vapor particles start to cool down because there are more of them, and they start to form tiny ice crystals.

Now, when the wind blows, it helps to push these tiny ice crystals down to the ground. As they fall, they hit other water droplets in the air and become bigger and bigger until they become too heavy to stay in the air anymore. That's when we get rain!

So, in simple terms, it rains when the cloud gets too full of water droplets that have turned into ice crystals, and the wind helps to push them down to the ground. And that's why we say, "Raindrops keep falling on my head."

**Response that is now received instead**
Imagine rain is like a bunch of tiny water balloons falling from the sky.
Clouds are like giant bags holding all these water balloons.
When the bags get too full, the balloons pop, and it starts to rain!

**Please, confirm that you have tested your contribution with lab generate and ensured that it does not produce any warnings or errors:**
It seems to generate fine.

### CODE REVIEW

**Comment by n1hility** [comment]
Thank you for your contribution to InstructLab! Unfortunately, once a Github repo is made public, all open PRs are automatically closed since they link against a private repo. We have detected that your PR might have been one of the ones affected by this change. If you are still interested in contributing your improvement, please fill out the following short form by no later than May 3rd, and we will get back to you with the additional steps necessary once we have had time to assess the PRs of those still interested:

https://forms.gle/V7SrPPMZDo6iGDYu8
